cg/store/api/add.py

```diff
@@ -157,14 +162,15 @@ def add_family(
                 LOG.debug(f"{internal_id} already used - trying another id")
 
         priority_db = PRIORITY_MAP[priority]
-        new_family = self.Family(
+        new_case = self.Family(
```

Comment:
Would be nice to finally change into self.Case, but perhaps out of scope of this PR

Reply:
I agree to both